Source/VCGeneration/Split.cs

```diff
@@ -1120,14 +1123,14 @@ void FocusRec(int focusIdx, IEnumerable<Block> blocks, IEnumerable<Block> freeBl
         return s;
       }
 
-      public static List<Split> FocusAndSplit(VCGenOptions options, Implementation impl, Dictionary<TransferCmd, ReturnCmd> gotoCmdOrigins, VCGen par, bool splitOnEveryAssert)
+      public static List<Split> FocusAndSplit(VCGenOptions options, Implementation impl, ImplementationRun run, Dictionary<TransferCmd, ReturnCmd> gotoCmdOrigins, VCGen par, bool splitOnEveryAssert)
```

Comment:
`ImplementationRun` references an `Implementation`, so the `impl` parameter here can be removed.